src/utils/jwt-utils.js

```diff
@@ -20,12 +20,29 @@ const jwtUtil = {
       expiresIn: AUTH_TOKEN_EXPIRY_MS,
     })
   ),
-  encryptToken: _.wrap(AES.encrypt, (encrypt, token) =>
-    encrypt(token, ENCRYPTION_SECRET).toString()
-  ),
-  decryptToken: _.wrap(AES.decrypt, (decrypt, token) =>
-    decrypt(token, ENCRYPTION_SECRET).toString(CryptoJS.enc.Utf8)
-  ),
+  encryptToken: (token) => {
+    // NOTE: The iv here is 16 characters long.
+    // This is because we generate 8 random bytes (1 byte = 8 bits)
+    // but hex is 4 bits per character..
+    const iv = crypto.randomBytes(8).toString("hex")
+    const cipher = crypto.createCipheriv("aes-256-cbc", ENCRYPTION_SECRET, iv)
```

Comment:
```suggestion
    const cipher = crypto.createCipheriv("aes-256-gcm", ENCRYPTION_SECRET, iv)
```

Btw, why not AES-GCM? Not super necessary given the context, but has the advantage of preventing forged cipher texts as long as your IV is randomised.

Reply:
actually i didn't wanna do it cos i'm not super familiar with crypto (or security stuff) in general. 

i changed it here cos it was simple enuf tho pog